asset/quickstart/export-assets-bigquery/main.go

```diff
@@ -45,6 +45,7 @@ func main() {
 				BigqueryDestination: &assetpb.BigQueryDestination{
 					Dataset: dataset,
 					Table:   "test",
+					Force:   true,
```

Comment:
Are we sure we want this in the sample code? Do we need a comment?

Reply:
It is in the sample for the other two languages that have a sample for this (Java and Python). I can add a comment, but it would just be restating what the docs say for this field.

Reply:
Sounds good. Thanks!